modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -40,59 +41,61 @@ endif::[]
 **** xref:contact-methods-rhnsd.adoc[rhnsd]
 ***** xref:contact-methods-osad.adoc[OSAD]
 **** xref:contact-methods-pushssh.adoc[Push via SSH]
+// Registration overviev, clients and different options
+** xref:registration-overview.adoc[Client Registration]
 // SLE Client Registration
-** xref:registration-methods.adoc[Registration Methods]
-*** xref:registration-webui.adoc[Register with the Web UI]
-*** xref:registration-bootstrap.adoc[Register with Bootstrap]
-*** xref:registration-cli.adoc[Register on the Command Line]
+*** xref:registration-methods.adoc[Registration Methods]
+**** xref:registration-webui.adoc[Register with the Web UI]
+**** xref:registration-bootstrap.adoc[Register with Bootstrap]
+**** xref:registration-cli.adoc[Register on the Command Line]
 // SUSE Client Registration
-** xref:registration-overview-suse.adoc[SUSE Client Registration]
-*** xref:clients-sle.adoc[SUSE Linux Enterprise Clients]
-*** xref:clients-opensuse.adoc[openSUSE Clients]
-*** xref:clients-sleses.adoc[Expanded Support Clients]
+*** xref:registration-overview-suse.adoc[SUSE Client Registration]
```

Comment:
What do you think we remove "Client Registration" and we leave only the OS/Vendor name in all cases? In the end we are already inside the "Client registration" section.

Reply:
> What do you think we remove "Client Registration" and we leave only the OS/Vendor name in all cases? In the end we are already inside the "Client registration" section.

I'm glad you asked @juliogonzalez, I've been thinking along similar lines as I'm always in favour of simplifying and decluttering. I'll speak with the team and would in any case deal with this in separate PR.